plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/niassa/WorkflowAction.java

```diff
@@ -345,6 +345,8 @@ public boolean search(Pattern query) {
     return ini.values().stream().anyMatch(v -> query.matcher(v.toString()).matches())
         || (runAccession != 0 && query.matcher(Integer.toString(runAccession)).matches())
         || limsKeysCollection.matches(query)
+        || workflowAccessions()
+            .anyMatch(workflow -> query.matcher(Long.toString(workflow)).matches())
```

Comment:
(probably more accurate to say workflow run, but it's internal so doesn't really matter)

Reply:
No, this is actually the workflow SWID, not the run SWID.

Reply:
...so it is. My mistake!